features/step_definitions/operator.rb

```diff
@@ -9,3 +9,125 @@
   })
   step %Q/the step should succeed/
 end
+
+Given /^etcd operator "([^"]*)" is removed successfully from "([^"]*)" project$/ do | name, proj_name|
+  ensure_admin_tagged
+  raise "!!! Error doesn't exist etcd operator: #{name} in this #{proj_name} project" unless subscription("#{name}").exists?
+  step %Q/I run the :delete client command with:/, table(%{
+    | object_type       | subscription |
+    | object_name_or_id | #{name}      |
+    | n                 | #{proj_name} |
+  })
+  raise "Error removing subscription: #{name} in #{proj_name} project" unless @result[:success]
+  step %Q/I ensures "#{name}" subscription is deleted from the "#{proj_name}" project after scenario/
```

Comment:
Why do we delete with delete command first and the register a clean-up step? Could you explain the intention?

Reply:
Sorry, my mistake. I just want to delete it immediately.